tasks/adventure_solutions/task_7_while.py

```diff
@@ -53,7 +55,7 @@ def dead(death_message):
     else:
         dead("the vampire is faster")
 
-elif door == "2":
+else:
```

Comment:
Then for any other selection it will go here. That will break the `door_greetings[door]` below